getting-started-gradle/src/test/java/unit/quickstart/GreetingResourceTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ *==========================================================================
+ *                      THIS FILE AND ITS CONTENTS ARE THE
+ *                    EXCLUSIVE AND CONFIDENTIAL PROPERTY OF
+ *
+ *                          EXPRETIO TECHNOLOGIES, INC.
+ *
+ * Any unauthorized use of this file or any of its parts, including, but not
+ * limited to, viewing, editing, copying, compiling, and distributing, is
+ * strictly prohibited.
+ *
+ * Copyright ExPretio Technologies, Inc., 2019. All rights reserved.
+ *=============================================================================
```

Comment:
Hmmm. This won't fly.

Reply:
Woops, sorry forget to switch context in IDE here.
I'm gonna work on this during the weekend.